app/test_main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+from unittest import mock
+from app.main import can_access_google_page
+import pytest
+
+
+@pytest.fixture()
+def mock_valid_google_url():
+    with mock.patch("app.main.valid_google_url") as mock_url:
+        yield mock_url
+
+
+@pytest.fixture()
+def mock_has_internet_connection():
+    with mock.patch("app.main.has_internet_connection") as mock_connection:
+        yield mock_connection
+
+
+def test_if_all_false(
+        mock_valid_google_url,
+        mock_has_internet_connection
+):
+    mock_valid_google_url.return_value = False
+    mock_has_internet_connection.return_value = False
+    assert can_access_google_page("www.google.com/") == "Not accessible", \
```

Comment:
Avoid using `\`. Use `( )`. Remake everywhere.

Reply:
you didn't fix `\` it here